Assessment.md

```diff
@@ -34,12 +34,18 @@ func startAssessmentWasPressed(){
     do{
         let userData = UserData(gender: .Female, birthday: Date()) // This is optinal if not provided the SDK will requst from the user his age and gender.
         // Start assessment
-        try SMKitUIModel.startAssessmet(viewController: self, type: AssessmentTypes.Fitness, userData: userData, delegate: self, onFailure: { error in
+        try SMKitUIModel.startAssessmet(
+        viewController: self,
+        type: AssessmentTypes.Fitness,
+        customAssessmentID // This is optinal if you have multiple 'Custom Assessment' you can provide the assessment ID in order to start the assessment
```

Comment:
@SencyNetanel shouldn't it be like?
customAssessmentID: <id>(String)

please split the example implementation into two. one for fitness type, and the other for custom type